aspnetcore/tutorials/web-api-javascript.md

```diff
@@ -40,11 +40,13 @@ The simplest `fetch` call accepts a single parameter representing the route. A s
 
 1. Create a *wwwroot* directory in the project root.
 
+1. Create a *js* directory inside of the *wwwroot* directory.
```

Comment:
Create  a *wwwroot/js*  folder to contain the JavaScript assets.

Reply:
I'd consider global replace directory with folder.

Reply:
Discussed at https://github.com/aspnet/AspNetCore.Docs/pull/12708#discussion_r290829685.

Reply:
I meant in this doc

Reply:
Yeah ... just mentioning that in passing. His reasoning is there, and I selected some quotes from guidance. If you want a firm rule followed, let me know. I would need to sweep the topics that I wrote. I used "folder" most of the time.

Reply:
No need to sweep. I prefer folder. Let's standardise new work on folder.

Reply:
Sometimes directory sounds better. In this case, folder is shorter and sounds better.